download/oci_download_unavailable.go

```diff
@@ -0,0 +1,67 @@
+//go:build no_oci
+
+package download
+
+import (
+	"context"
+	"github.com/open-policy-agent/opa/bundle"
+	"github.com/open-policy-agent/opa/plugins/rest"
+)
+
+
+// NewOCI returns a new Downloader that can be started.
+func NewOCI(config Config, client rest.Client, path, storePath string) *OCIDownloader {
```

Comment:
[nit]
```suggestion
func NewOCI(Config, rest.Client, string, string) *OCIDownloader {
```

Perhaps we can also get rid of the comments.